lib/commands/settings.js

```diff
@@ -68,7 +68,7 @@ commands.setSourceTreeFilter = async function (filter) {
     throw new errors.UnknownCommandError('argument must be numeric and larger than zero');
   }
   if (result.status === youiEngineDriverReturnValues.WEBDRIVER_INVALID_SELECTOR) {
-    throw new errors.InvalidSelectorError("Attribute filter should have following format: [@attributeType='attributeValue']");
+    throw new errors.InvalidSelectorError("Attribute filter should have following format: [@attributeType='attributeValue'] or '' to reset. ('' is supported in You.i Engine 5.2+)");
```

Comment:
How does this error work? Sorry, just want to be clear because the error message is kind of hard to understand.

Reply:
The error is thrown by the engine if the format is incorrect.